src/Microsoft.DotNet.ImageBuilder/src/Commands/CleanAcrImagesCommand.cs

```diff
@@ -79,6 +83,10 @@ public override async Task ExecuteAsync()
                     await ProcessManifestsAsync(acrClient, acrContentClient, deletedImages, deletedRepos, repository,
                         manifest => !manifest.Tags.Any() && IsExpired(manifest.LastUpdatedOn, Options.Age));
                     break;
+                case CleanAcrImagesAction.PruneEol:
+                    await ProcessManifestsAsync(acrClient, acrContentClient, deletedImages, deletedRepos, repository,
+                        manifest => manifest.Tags.Any() && HasExpiredEol(manifest, Options.Age));
```

Comment:
We want _any_ image to be eligible for deletion, not just tagged images.

```suggestion
                        manifest => HasExpiredEol(manifest, Options.Age));
```

Reply:
I was trying to avoid annotation manifests. Is there a better way of doing that?

Won't untagged images be deleted during cleanup of dangling images?

Reply:
> I was trying to avoid annotation manifests. Is there a better way of doing that?

See my changes in https://github.com/dotnet/docker-tools/pull/1378

> Won't untagged images be deleted during cleanup of dangling images?

No, we don't have a policy for cleaning dangling images in `public/*` repos, which is what this action type would be applied to.

Reply:
Fixed with https://github.com/dotnet/docker-tools/pull/1379/commits/38aa3f8b50aed5c3b85368216ddcbe1c9e4b5599